index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Ali Ahmad Karimi Portfolio</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100&display=swap" rel="stylesheet">
```

Comment:
Please know that when you imported the font you only selected one style which has a weight of 100 🙃, this caused your website to use the weight 100 for every text you have even if you specify other weights on your CSS file.

To fix this there are two solutions:

1. You can either select specific styles when importing, for example in your case you'll need the styles that have weights of 400, 500, and 700 for this specific project.

2. Or you can just use this line to import all styles instead and control the weight from your CSS file:  `<link href="https://fonts.googleapis.com/css2?family=Poppins" rel="stylesheet">` I removed the `:wght@100&display=swap` part of the link.

This should fix some styling issues for your website that I mentioned in other comments! 😁😁